### PR TITLE
contract admin cleanup

### PR DESCRIPTION
**Pull request by Joeysantoro**

### CODE REVIEW

File: contracts/pcv/convex/VotiumBriber.sol [hunk 1]
```diff
@@ -43,7 +43,7 @@ contract VotiumBriber is CoreRef {
     /// the _proposal ID, if _choiceIndex is out of range, or of block.timestamp
     /// is after the deadline for bribing (usually 6 hours before Convex snapshot
     /// vote ends).
-    function bribe(bytes32 _proposal, uint256 _choiceIndex) public onlyGovernorOrAdmin whenNotPaused {
+    function bribe(bytes32 _proposal, uint256 _choiceIndex) public onlyTribeRole(TribeRoles.VOTIUM_ROLE) whenNotPaused {
```
Comment (ditchfieldcaleb) [inline contracts/pcv/convex/VotiumBriber.sol:46]: should this be `oneOfTwoRoles(TribeRoles.GOVERNOR, TribeRoles.VOTIUM_ROLE)` or was removing the gov intentional?

Comment (Joeysantoro) [reply contracts/pcv/convex/VotiumBriber.sol]: removing gov is fine, its only ever been called by the ops timelock

**Review by ditchfieldcaleb — APPROVED** [review]

**Review by thomas-waite — COMMENTED** [review]
Looks good to me. Do we also want to deploy this contract and deprecate the old one? We could make a TC proposal to do it

**Comment by Joeysantoro** [comment]
> Looks good to me. Do we also want to deploy this contract and deprecate the old one? We could make a TC proposal to do it

I don't think this is necessary. It doesn't change anything practically

**Pull request merged**